extension/src/plots/model/index.ts

```diff
@@ -91,6 +91,9 @@ export class PlotsModel extends ModelWithPersistence {
     )
     this.comparisonOrder = this.revive(PersistenceKey.PLOT_COMPARISON_ORDER, [])
     this.customPlotsOrder = this.revive(PersistenceKey.PLOTS_CUSTOM_ORDER, [])
+
+    this.cleanupOutdatedCustomPlotsState()
```

Comment:
How long should we wait before we remove these functions and delete the metrics persistence keys?

Reply:
@mattseddon was the one to make the comment, so I'll leave the verdict to him. But do we even need to migrate the data? I'd just delete it and start fresh.

Reply:
> @mattseddon was the one to make the comment, so I'll leave the verdict to him. But do we even need to migrate the data? I'd just delete it and start fresh.

Personally, I vote we migrate. It's not that difficult to do and it ensures users who have already created custom plots doesn't lose anything saved when they get the latest release. Though, on the other hand, I doubt that many users have created custom plots so no strong preference 🤔

Reply:
Up to you. Please set a reminder for one month from now to turn this off.